.env.example

```diff
@@ -146,8 +146,13 @@ ENABLE_LIGHTWAVE=True
 # In-built django variable that logouts user if cookie expiration time has been exceeded
 SESSION_COOKIE_AGE=43200
 
+#### Auto rejection of credentialing applications Start ####
```

Comment:
I thought it might be a good idea to make a block of settings for autorejection and group them together